06_gpu_and_ml/flan_t5/finetune.py

```diff
@@ -0,0 +1,246 @@
+# # Finetuning Flan-T5
+#
+# Example by [@anishpdalal](https://github.com/anishpdalal)
+#
+# [Flan-T5](https://huggingface.co/docs/transformers/model_doc/flan-t5) is a highly versatile model that's been instruction-tuned to
+# perform well on a variety of text-based tasks such as question answering and summarization. There are smaller model variants available which makes
+# Flan-T5 a great base model to use for finetuning on a specific instruction dataset with just a single GPU. In this example, we'll
+# finetune Flan-T5 on the [Extreme Sum ("XSum")](https://huggingface.co/datasets/xsum) dataset to summarize news articles.
+
+# ## Defining dependencies
+#
+# The example uses the `dataset` package from HuggingFace to load the xsum dataset. It also uses the `transformers`
+# and `accelerate` packages with a PyTorch backend to finetune and serve the model. Finally, we also 
+# install `tensorboard` and serve it via a web app. All packages are installed into a Debian Slim base image 
+# using the `pip_install` function.
+#
+
+from modal import (
+    Image,
+    method,
+    Stub,
+    SharedVolume,
+    wsgi_app
+)
+
+from pathlib import Path
+
+VOL_MOUNT_PATH = Path("/vol")
+
+# Other Flan-T5 models can be found [here](https://huggingface.co/docs/transformers/model_doc/flan-t5)
+BASE_MODEL = "google/flan-t5-base"
+
+image = Image.debian_slim().pip_install(
+    "accelerate",
+    "transformers",
+    "torch",
+    "datasets",
+    "tensorboard",
+)
+
+stub = Stub(name="example-news-summarizer", image=image)
```

Comment:
Can you change the name of the example to:

```suggestion
stub = Stub(name="example-fine-tuning-flan-t5", image=image)
```